### PR TITLE
Add fiat value for tokens

### PR DESCRIPTION
**Pull request by CedrikNikita**

### CODE REVIEW

**Comment by Liubov-crypto** [comment]
LGTM in general. I can see fiat value for tokens in token list and on token details page. 

One question: will we add fiat value to the transactions list and Tx details page? or leave as it is? 


https://github.com/user-attachments/assets/b7e80191-e49f-40c7-bfe6-f10ed552f093

**Review by peronczyk — CHANGES_REQUESTED** [review]

File: src/protocols/aeternity/libs/AeternityAdapter.ts [hunk 1]
```diff
@@ -259,17 +262,26 @@ export class AeternityAdapter extends BaseProtocolAdapter {
...
     try {
       const tokens: ITokenBalanceResponse[] = camelCaseKeysDeep(await fetchAllPages(
         () => fetchFromMiddleware(`/v2/aex9/account-balances/${address}?limit=100`),
         fetchFromMiddleware,
       ));
+      if (tokens.length) {
+        await Promise.race(
+          [watchUntilTruthy(areTokenSwapsReady),
+            new Promise((resolve) => setTimeout(resolve, 5000)),
+          ],
```
Comment (peronczyk) [inline src/protocols/aeternity/libs/AeternityAdapter.ts:275]:
```suggestion
          [
            watchUntilTruthy(areTokenSwapsReady),
            sleep(5000),
          ],
```

File: src/protocols/aeternity/libs/AeternityAdapter.ts [hunk 2]
```diff
@@ -145,6 +146,8 @@ export class AeternityAdapter extends BaseProtocolAdapter {
       name: this.coinName,
       symbol: this.coinSymbol,
       convertedBalance,
+      // This price is in coin per token
```
Comment (peronczyk) [inline src/protocols/aeternity/libs/AeternityAdapter.ts:149]: This comment is not necessary as the interface has this property already commented. I suggest to use DocBlock there to have IDE tooltips with the comment.

File: src/popup/pages/Assets/AssetDetailsInfo.vue [hunk 3]
```diff
@@ -217,20 +270,51 @@ export default defineComponent({
...
+        ? tokenSwaps.value
+          .find((token) => token.address === sharedAssetDetails.tokenData.contractId)
+        : undefined
+    ));
+
+    const adapter = computed(() => ProtocolAdapterFactory.getAdapter(
+      sharedAssetDetails.tokenData.protocol,
+    ));
+
+    const assetData = computed(() => ({
```
Comment (peronczyk) [inline src/popup/pages/Assets/AssetDetailsInfo.vue:292]: I suggest to type this computed property.

**Pull request closed without merging**